src/python/T0/RunConfig/RunConfigAPI.py

```diff
@@ -553,18 +554,18 @@ def configureRunStream(tier0Config, run, stream, specDirectory, dqmUploadProxy):
             taskName = "Repack"
 
             if tier0Config.Global.EnableUniqueWorkflowName:
-                workflowName = "Repack_Run%d_Stream%s_%s_v%s_%s" % (run, stream, 
-                    tier0Config.Global.AcquisitionEra, streamConfig.Repack.ProcessingVersion, 
+                workflowName = "Repack_Run%d_Stream%s_%s_v%s_%s" % (run, stream,
+                    tier0Config.Global.AcquisitionEra, streamConfig.Repack.ProcessingVersion,
                     time.strftime('%y%m%d_%H%M', time.localtime(time.time())))
             else:
                 workflowName = "Repack_Run%d_Stream%s" % (run, stream)
 
             specArguments = {}
 
-            if stream == 'ScoutingPF':
-                specArguments['Memory'] = 2000
+            if stream == setStreamMem(tier0Config,stream):
```

Comment:
with the current deffinition of `setStreamMem`, this will always evaluate to `True`. I don't think that is what you are trying to achieve.